Makefile

```diff
@@ -91,12 +91,26 @@ docker/%:
 	docker build -f $(MODULE)/Dockerfile -q -t docker.io/activiti/$(MODULE):$(RELEASE_VERSION) $(MODULE)
 	docker push docker.io/activiti/$(MODULE):$(RELEASE_VERSION)
 
+# follow instructions at https://github.com/docker/hub-feedback/issues/496#issuecomment-277562292
 docker-delete/%:
 	$(eval MODULE=$(word 2, $(subst /, ,$@)))
 
 	@echo "Delete image from Docker Hub for $(MODULE):$(RELEASE_VERSION)..."
-	curl --silent --show-error --fail -X DELETE -u "${DOCKERHUB_USERNAME}:${DOCKERHUB_PASSWORD}" \
+	$(eval TOKEN=$(shell curl --silent --show-error --fail \
+		-X POST \
+		-H "Content-Type: application/json" \
+		-H "Accept: application/json" \
+		-d '{"username":"${DOCKERHUB_USERNAME}","password":"${DOCKERHUB_PASSWORD}"}' \
+		https://hub.docker.com/v2/users/login/ | jq ".token"))
+	-@curl --silent --show-error --fail \
+		-X DELETE \
+		-H "Authorization: JWT $(TOKEN)" \
 		https://hub.docker.com/v2/repositories/activiti/$(MODULE)/tags/$(RELEASE_VERSION)
```

Comment:
with the leading `-` it will ignore the exit code of curl - thus not failing when the tag removal is failing - is that really intended?

Reply:
no indeed, what I wanted was to "prevent the token from being reused" by logging out even if the deletion failed: any idea how to achieve that?

Reply:
something like https://stackoverflow.com/a/62194313/547195 to actually execute multiple commands in the same shell so you can always do the logout and then exit depending on the first curl exit

Reply:
done :+1: